flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/lineage/ModifyType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.lineage;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/**
+ * The existing `ModifyType` should be marked as `PublicEvolving` and users can get it from table
```

Comment:
I find this comment misleading. If there is another modifyType - then this should be referred to with a link in the javadoc. Is the should a TODO or no longer needed? I see SinkModifyOperation has an identical modifyType which I assume you are referring to. Can we not use this enum / refactor to one enum?

Reply:
Can't agree more.